10.Mender-server/docs.md

```diff
@@ -7,6 +7,67 @@ shortcode-core:
 github: false
 ---
 
+## Mender 3.7.2
+
+_Released 02.12.2024_
+
+### Statistics
+
+| Developers with the most changesets | |
+|---|---|
+| Lluis Campos | 3 (42.9%) |
+| Roberto Giovanardi | 1 (14.3%) |
+| Fabio Tranchitella | 1 (14.3%) |
+| Ole Petter Orhagen | 1 (14.3%) |
+| Alf-Rune Siqveland | 1 (14.3%) |
+
+| Developers with the most changed lines | |
+|---|---|
+| Fabio Tranchitella | 62 (42.5%) |
+| Ole Petter Orhagen | 45 (30.8%) |
+| Lluis Campos | 28 (19.2%) |
+| Alf-Rune Siqveland | 10 (6.8%) |
+| Roberto Giovanardi | 1 (0.7%) |
+
+| Top changeset contributors by employer | |
+|---|---|
+| Northern.tech | 6 (85.7%) |
+| ole@orhagen.no | 1 (14.3%) |
```

Comment:
@oleorhagen thinks that he is an external contributor already :broken_heart: 

@tranchitella, to correct the statistics we need to add his personal e-mail to [this file](https://github.com/mendersoftware/integration/blob/master/extra/gitdm/group-maps/Northern.tech) and then rerun the command.

Reply:
Done here: https://github.com/mendersoftware/integration/pull/2567